source/renderer/app/components/widgets/forms/InlineEditingInput.js

```diff
@@ -301,11 +301,9 @@ export default class InlineEditingInput extends Component<Props, State> {
           type="text"
           maxLength={maxLength}
           label={label}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
           onChange={this.onChange}
           onKeyDown={(event) => this.handleInputKeyDown(event)}
-          error={isActive ? error : !!error}
```

Comment:
@aleksandardjordjeviciohk please check the impact of this.

Reply:
Done